graphql-server/pkg/dataloaders/proposal.go

```diff
@@ -18,10 +21,27 @@ type ProposalTallyResultDataloader interface {
 }
 
 type IProposalDataloader struct {
-	proposalTallyResultLoader			   ProposalTallyResultDataloader
+	proposalLoader            *ProposalLoader
+	proposalTallyResultLoader ProposalTallyResultDataloader
 }
 
 func NewProposalDataloader(proposalQuery queries.IProposalQuery) ProposalDataloader {
+	proposalLoader := &ProposalLoader{
+		maxBatch: 1000,
```

Comment:
Shouldn't hard code here. It should be a package variable.

Reply:
By package variable, do you mean configuration values loaded from environment variables when starting the server (like the database urls)?

If so, would that mean all dataloaders in the project will share the same `maxBatch` value?

Reply:
Too much for configuration, just package variable is good enough IMO.

Reply:
oh is this a thing for golang, thought configs would be from a single source